src/config/config.js

```diff
@@ -6,13 +6,13 @@ const Config = {
   newVersionUrl: process.env.REACT_APP_NEW_VERSION_URL || 'https://gdlr.info/newversion',
   mnemonicToSeed: process.env.REACT_APP_MNEMONIC_TO_SEED || true,
   logLevel: process.env.REACT_APP_LOG_LEVEL || 'debug',
-  serverUrl: process.env.REACT_APP_SERVER_URL || 'http://localhost:3003',
-  gunPublicUrl: process.env.REACT_APP_GUN_PUBLIC_URL || 'http://localhost:3003/gun',
+  serverUrl: 'https://good-server.herokuapp.com',
```

Comment:
revert all modifications for this file
check how to correctly add .env file to detox
@alexpirogovqa